.github/workflows/auto-fix-lint.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         with:
           ref: ${{ github.head_ref }}
           token: ${{ secrets.GH_CLOUD_SDK_JS_ADMIN_WRITE_TOKEN }}
+          node-version: ${{ vars.DEFAULT_NODE_VERSION }}
```

Comment:
Seems to still fail, do we need to provide 22 value here?

Reply:
Ignore my previous comment, it picks up the correct workflow file, but maybe not the updated actions file?

Reply:
I will change the action tag to see if it works before merging. But you are right, it is due to the `@main` tag.